### DISCUSSION TITLE
Mono libs not found

### DISCUSSION THREAD

**Discussion by TheoMarque** [mbround18/valheim-docker#709] [Help!]
Run fresh install, just git clone your repository without change anything and change only type server into ValheimPlus

```
valheim-docker-valheim-1  | Fallback handler could not load library /home/steam/valheim/valheim_server_Data/Mono/data-0x55602eacfb40.so
valheim-docker-valheim-1  | Fallback handler could not load library /home/steam/valheim/valheim_server_Data/Mono/mscorlib/mono/aot-cache/amd64/data-0x55602eacfb40.so
```

spam in console, server won't run.
Why? because in container does not contain any mono libs in this path. 

<img width="489" alt="image" src="https://github.com/mbround18/valheim-docker/assets/19385639/dcec2fc8-52ce-480d-b2e0-6367a12b4b14">

Image can run only vanilla type.

**Comment by mbround18** [comment]
@TheoMarque ValheimPlus has been having a few issues as they havent updated in a while. I recommend BepInEx or BepInExFull for the mod type to use mods.

**Reply by TheoMarque** [reply]
Used Grantapher V+ mod [https://github.com/Grantapher/ValheimPlus](https://github.com/valheimPlus/ValheimPlus), but in other than yours image works fine. So a problem is directly in your image suing other than previous version of valheim plus. Use this repo as default, valheimplus old repo is abandoned.
